src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Monsters/Silver/TheThingInTheSwamp.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Alsein.Extensions;
+
+namespace Cynthia.Card
+{
+    [CardEffectId("70086")]
+    public class TheThingInTheSwamp : CardEffect
+    {//将墓场3张铜色/银色单位牌放回牌组。
+        public TheThingInTheSwamp(GameCard card) : base(card) { }
+        public override async Task<int> CardPlayEffect(bool isSpying, bool isReveal)
+        {
+            //参见issue#22，这里使用showcardmove
+            var list = Game.PlayersCemetery[PlayerIndex].Where(x => (x.Status.CardId == "24028"));
+            var result = await Game.GetSelectMenuCards(Card.PlayerIndex, list.ToList(), 3, "拣选并放回最多三颗果子");
+            foreach (var card in result)
+            {
+                card.Effect.Repair();
+                await Game.ShowCardMove(new CardLocation(RowPosition.MyDeck, RNG.Next(0, Game.PlayersDeck[Card.PlayerIndex].Count)), card);
+            }
+            await Game.GameRowEffect[AnotherPlayer][Card.Status.CardRow.MyRowToIndex()].SetStatus<ImpenetrableFogStatus>();
```

Comment:
In the card description, it first spawn fog, and then move the cards to deck, which is inconsistent with the code here. Which one is your intention?

Reply:
The coded effect is the intended effect: put back any number of foglets and then spawn fog to the opposing row, it is meant to allow you to summon a foglet even there are none left in the graveyard. I updated translation files to reflect that.

Reply:
Sounds good. Thanks!